SCClassLibrary/Common/Control/Server.sc

```diff
@@ -1085,9 +1085,10 @@ Server {
 
 		addr.sendMsg("/quit");
 
-		if(watchShutDown and: { this.unresponsive }) {
+		if(this.unresponsive) {
 			"Server '%' was unresponsive. Quitting anyway.".format(name).postln;
-			watchShutDown = false;
+			pid !? {thisProcess.platform.killProcessByID(pid)};
```

Comment:
i think this should be turned into an `if` using pid.notNil as a condition; in the true branch kill by PID and post a more forceful message (like "Server '%' was unresponsive. Killing forcefully.") and in the false branch post the original message.